estate/models/estate_property.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare, float_is_zero
+from dateutil.relativedelta import relativedelta
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Properties"
+    
+    _sql_constraints = [
+        ('check_prices', 'CHECK(expected_price > 0 AND selling_price >= 0)',
+         "Prices should be positive. Additionnaly, expected price can't be 0.")
+    ]
+    
+    # Metadata
+    name = fields.Char('Title', required=True, translate=True, default="Best House In Town")
+    active = fields.Boolean('Active', default=True)
```

Comment:
By default, the label will be the name of the field capitalize. In this case, it will be `Active` so there is no need to define it.